internal/localizer/i18n/config/ecode/en.yaml

```diff
@@ -108,6 +108,14 @@ E2023:
   desc: KeyedList key not unique
   text: KeyedList key "{{.Key}}" duplicates
   help: ensure KeyedList's key is unique
+E2024:
+  desc: invalid version pattern
+  text: '"{{.Value}}" cannot be parsed into dotted-decimal format, {{.Error}}'
+  help: "available patterns: 255.255.255.255"
```

Comment:
For general use cases, change help to `ensure version pattern is: <MAJOR>.<MINOR>.<PATCH>[.<OTHER>]...`